attr.go

```diff
@@ -27,7 +27,8 @@ const (
 	AttrUnderline
 	AttrDim
 	AttrItalic
+	AttrStrikeout
```

Comment:
This will probably break on 32-bit platforms, where int is a 32-bit quantity.

In particular, I'm not sure what 1 << 31 on such platforms does -- it probably makes the number negative.

This may represent more risk of breakage than I want to take on.

In retrospect the decision to encode these attributes in the high order bits was probably a mistake, but it may have been influenced by how termbox did it.

Probably we should look at changing the base type of attribute to something that is a bit more future proof.

Having said that, this is also probably a breaking change for some users, so we might not not want to do that unless we're prepared to bump a version (per semver).

Reply:
You’re right, this is a problem. I did check the AttrStrikeout value and it wasn’t negative, so I wrongly assumed it was all good. I’m not experienced in this, so sorry about that.